### ISSUE TITLE
Change paper background color for all pages, Apply to All Pages

### ISSUE THREAD

**Issue by wltjr** [xournalpp/xournalpp#4120]
**Is your feature request related to a problem? Please describe.**
To change paper background color, one must change each page individually.

**Describe the solution you'd like**
It would be really nice to be able to change the background color of all pages, like changing the background style, having a option to "Apply to All Pages" would save a lot of time!

**Describe alternatives you've considered**
Going through page by page and changing the color manually, which is tedious and a page could be skipped/overlooked, and for large documents, really not practical.

**Additional context**
Basically, the same feature as on changing paper background.
